db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql

```diff
@@ -1 +1,17 @@
 \echo 'Upgrade Queries not required for transition from $CURRENT_VERSION to $UPGRADE_VERSION'
+
+-- Rollback script for master.app_detail
+ALTER TABLE master.app_detail ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.app_detail DROP CONSTRAINT pk_appdtl_id;
```

Comment:
We only drop the constraint, we should also add the PK constraint with the lang_code column back @dhanendra06